dandiapi/api/dashboard.py

```diff
@@ -59,6 +62,9 @@ def _non_valid_assets(self):
     def _uploads(self):
         return Upload.objects.annotate()
 
+    def _users(self):
+        return User.objects.select_related('metadata').order_by('-date_joined')
```

Comment:
is this restricted to active users?

Reply:
No, but I just added a column to also show the current status. Would you rather unapproved users not show up at all?

Reply:
yes. at least for my present needs, only approved users.